data/json/requirements/materials.json

```
@@ -11,6 +11,12 @@
     "//": "Materials used when forming bullets",
     "components": [ [ [ "lead", 1 ], [ "bismuth", 3 ] ] ]
   },
+  {
+    "id": "ammo_bullet_birdshot",
+    "type": "requirement",
+    "//": "Materials used when forming bullets for birdshot",
+    "components": [ [ [ "lead", 1 ], [ "bismuth", 3 ], [ "bb", 2 ] ] ]
```

Having the BBs first on the list should ensure that looted birdshot will uncraft into BBs instead of lead, screm at me if I'm wrong tho
```suggestion
    "components": [ [ [ "bb", 2 ], [ "lead", 1 ], [ "bismuth", 3 ] ] ]
```

---

I mean, the uncraft for normal birdshot is explicitly set anyways. But yeah, probably not a bad idea for the BP and 20 gauge varieties just in case